skate-plugin/project-gen/src/jvmMain/kotlin/slack/tooling/aibot/PainterResource.kt

```diff
@@ -28,7 +28,7 @@ import org.jetbrains.compose.resources.decodeToSvgPainter
 
 // Migration snippet copied from https://github.com/JetBrains/compose-multiplatform-core/pull/1457
 // To resolve deprecated painterResource function from upgrading compose-jb
-class PainterResource {
+object PainterResource {
```

Comment:
Does it need to be an object vs top-level functions?

Reply:
Ah yeah it doesn't need to be top-level. I wrapped it in the class originally because I was afraid it would call the deprecated painterResource function, but since I removed the import it should be good. I think I should remove the object :
d432093d8eb493b111c741a093ee93b0bdd59d71